docs/reference/components.cue

```diff
@@ -739,6 +739,18 @@ components: {
 				}
 			}
 
+			_timezone: {
+				common:      false
+				description: "The name of the time zone to apply to timestamp conversions that do not contain an explicit time zone. This overrides the global [`timezone` option][docs.reference.configuration.global-options#timezone]"
```

Comment:
Would it be possible to provide more info about what are valid specifiers? We could perhaps link to https://docs.rs/chrono-tz/0.5.3/chrono_tz/enum.Tz.html or https://www.iana.org/time-zones

Reply:
Yes, that would be useful.